site/data/qasida/ya-nasim-al-ward.json

```
@@ -5,34 +5,61 @@
     },
     "verses": {
         "ara": [
-            ["يَا نَسِيمَ الوَرْدِ خَبِّرْ لِلرَشَا"],
-            ["لَم يَزِدْنِي الوِرْدُ إلَا عَطَشَا"],
+            ["لَا إِلَهَ إِلاَّ اللهْ اللهْ اللهْ"],
+            ["لَا إِلَهَ إِلَّا اللَّهُ جُدْ عَلَيْنَا"],
 
-            ["لِي حَبِيبٌ حُبُهُ وَسْطَ الحَشَا"],
-            ["إن يَشَأ يَمْشِي عَلَى خَدِّي مَشَا"],
+            ["يَا نَسِيْمَ الْوَرْدِ خَبِّرْ لِلرَّشَا"],
+            ["لَمْ يَزِدْنِي الْوِرْدُ إِلَّا عَطَشَا"],
 
-            ["قَولُهُ قَولِي وَقَولِي قَولُهُ"],
-            ["إن يَشَأ شِئتُ وَإن شِئتُ يَشَا"],
+            ["لِيْ حَبِيبٌ حُبُّهُ حَشْوُ الْحَشَا"],
+            ["إِنْ يَشَأْ يَمْشِي عَلَى خَدِّيْ مَشَى"],
 
-            ["رُوحُهُ رُوحِي وَرُوحِي رُوحُهُ"],
-            ["إن عَاشَا عِشْتُ وَإن عِشْتُ عَاشَا"]
+            ["(قَوْلُهُ قَوْلِي وَقَوْلِي قَوْلُهُ)"],
+            ["إِنْ يَشَأْ شِئْتُ وَإِنْ شِئْتُ يَشَا"],
+
+            ["رُوحُهُ رُوْحِيْ وَرُوْحِيْ رُوحُهُ"],
+            ["(إِنْ عَشَاعِشْتُ وَإِنْ عِشْتُ عَشَا)"],
```

```suggestion
            ["(إِنْ عَشَا عِشْتُ وَإِنْ عِشْتُ عَشَا)"],
```
(Added a space between the words)